src/piyushPJun20/ItemId.java

```diff
@@ -0,0 +1,31 @@
+package piyushPJun20;
+
+public class ItemId {
+	void getItemID(String input) {
+		String itemId = "";
+		if (input.contains("itemId")) {
+			String temp = input.substring(input.indexOf("itemId"));
+			for (int index = 0; index < temp.length(); index++) {
+				if (temp.charAt(index) >= '0' && temp.charAt(index) <= '9') {
+					itemId += temp.charAt(index);
+					if (!(temp.charAt(index + 1) >= '0' && temp.charAt(index + 1) <= '9'))
```

Comment:
This if is not required. You should write else of above if block.
if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}

Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}